recipes/cycle_finder/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "1.0.0" %}
+{% set sha256 = "1254695585e1211b279ca036ac24594100ef1d21998e9fe1739f83364fbb7942" %}
+
+
+package:
+  name: cycle_finder
+  version: {{ version }}
+
+source:
+  sha256: '{{sha256}}'
+  url: https://github.com/rkajitani/cycle_finder/archive/refs/tags/v1.0.0.tar.gz
```

Comment:
You could use here the version token, this will help our bit to auto update the recipe with every new version